.github/workflows/main.yml

```diff
@@ -280,12 +201,6 @@ jobs:
       matrix:
         group: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20]
     steps:
-      - uses: actions/cache@v2
-        id: restore-build
-        with:
-          path: ./*
-          key: ${{ runner.os }}-${{ github.sha }}
```

Comment:
we can't remove this section from the jobs because it restores the actual built packages which are needed for the tests

Reply:
Good catch @flybayer! The moment you need to use `actions/cache` for other jobs (like the one you pointed for the `test` jobs) I think it makes the migration to `actions/setup-node`'s `cache` option not worth it. `setup-node` uses `actions/cache` internally when enabling `cache` option.

Unless you see a clean way to use `cache` option in `setup-node` and test jobs can use packages from that cache folder (the default one), we should close this issue.

What do you think @flybayer

Reply:
> Unless you see a clean way to use cache option in setup-node and test jobs can use packages from that cache folder (the default one), we should close this issue.

Yeah, I think that makes sense to close it. Thanks for your effort though, we appreciate you looking into this!